lib/ffi/data_converter.rb

```diff
@@ -0,0 +1,37 @@
+module FFI
```

Comment:
Should I add a license header? If so, what copyright line should I use?

Reply:
Follow whatever pattern other .rb files use. We can discuss sorting out licensing, headers, and copyrights in a separate issue if necessary.

Reply:
I used the same as in `lib/ffi/errno.rb`.